lib/api/repair/v1/service.test.ts

```diff
@@ -0,0 +1,87 @@
+import { config } from "@mtfh/common/lib/config";
+import { useAxiosSWR } from "@mtfh/common/lib/http";
+import { useRepairs } from "./service";
+
+import { RepairsFilters } from "./types";
+
+jest.mock("@mtfh/common/lib/http", () => ({
+  ...jest.requireActual("@mtfh/common/lib/http"),
+  useAxiosSWR: jest.fn(),
+}));
+
+const repairsMock = [
+  {
+    reference: 10036383,
+    dateRaised: "2021-11-05T17:38:21.748934Z",
+    lastUpdated: null,
+    priority: "5 [N] NORMAL",
+    property: "339-428 Fellows Court  Weymouth Terrace",
+    propertyPostCode: "E2 8LB",
+    owner: "HH General Building Repai",
+    description: "ELC - 5 lights on balcony walkway on 9th floor are out. ",
+    propertyReference: "00075623",
+    tradeCode: "EL",
+    tradeDescription: "Electrical - EL",
+    status: "In Progress",
+  },
+];
+
+test("useRepairs: it should send the correct query params to the API", async () => {
+  const id = "00075623";
+  (useAxiosSWR as jest.Mock).mockResolvedValueOnce(repairsMock);
+  const response = await useRepairs(id);
```

Comment:
Should probably use `@testing-library/react-hooks` for testing hooks.

Reply:
Updated to use `@testing-library/react-hooks` for testing hooks